pkgs/top-level/all-packages.nix

```diff
@@ -18864,6 +18863,12 @@ with pkgs;
     headless = true;
   });
 
+  virtualboxExtpack = callPackage ../applications/virtualization/virtualbox/extpack.nix { };
+
+  virtualboxWithExtpack = lowPrio (virtualbox.override {
```

Comment:
why do we want to have the `lowPrio` here? Just to make it more deterministic if somebody installs both `virtualbox` and `virtualboxWithExtpack` which one will win?

Reply:
@flokli Thanks for reviewing this!

To be honest, I'm actually not sure what role `lowPrio` is playing.

You can't tell because Github cuts off the `all-packages.nix` file, but there are actually a couple different versions of the virtualbox derivation:

```nix
  virtualbox = callPackage ../applications/virtualization/virtualbox {
    stdenv = stdenv_32bit;
    inherit (gnome2) libIDL;
    pulseSupport = config.pulseaudio or true;
  };

  virtualboxHardened = lowPrio (virtualbox.override {
    enableHardening = true;
  });

  virtualboxHeadless = lowPrio (virtualbox.override {
    enableHardening = true;
    headless = true;
  });
```

I setup `virtualboxWithExtpack` to use `lowPrio` because that's what `virtualboxHardened` and `virtualboxHeadless` derivations were doing.

I'm not sure if it is "correct" to use `lowPrio` here or not.

If you think that `lowPrio` shouldn't be used here, definitely let me know!